challenges/Python_Conda_Basics/README.md

```diff
@@ -0,0 +1,223 @@
+Building from source results in a longer installation time for packages, so you may need to wait a few minutes for the install to finish.
+
+Congratulations, you have built NumPy from source in your conda environment!
+However, we did not link in any additional linear algebra packages, so this version of NumPy is not optimized.
+Let's install a more optimized version using a different method instead, but first we must uninstall the pip-installed NumPy:
+
+```
+$ pip uninstall numpy
+```
+
+The traditional, and more basic, approach to installing/uninstalling packages into a conda environment is to use the commands `conda install` and `conda remove`.
+Installing packages with this method checks the [Anaconda Distribution Repository](https://docs.anaconda.com/anaconda/packages/pkg-docs/) for pre-built binary packages to install.
+Let's do this to install NumPy:
+
+```
+$ conda install numpy
+```
+
+Because NumPy depends on other packages for optimization, this will also install all of its dependencies.
+Congratulations, you have just installed an optimized version of NumPy, now let's test it!
+
+## Testing your new environment
+
+Let's run a small script to test that things installed properly.
+Since we are running a small test, we can do this without having to run on a compute node. 
+
+> NOTE: Remember, at larger scales both your performance and your fellow users' performance will suffer if you do not run on the compute nodes.
+It is always highly recommended to run on the compute nodes (through the use of a batch job or interactive batch job).
+
+Make sure you're in the correct directory and execute the example Python script:
+
+```
+$ cd ~/hands-on-with-summit/challenges/Python_Conda_Basics/
+$ python3 hello.py
+
+Hello from Python 3.7.11!
+You are using NumPy 1.21.2
+```
+
+Congratulations, you have just created your own Python environment and ran on one of the fastest computers in the world!
+
+## Additional Tips
+
+* Cloning an environment:
+
+    It is not recommended to try to install new packages into the base environment.
+    Instead, you can clone the base environment for yourself and install packages into the clone.
+    To clone an environment, you must use the `--clone <env_to_clone>` flag when creating a new conda environment.
+    An example for cloning the base environment on Ascent is provided below:
+
+    ```
+    $ conda create -p /ccsopen/proj/<YOUR_PROJECT_ID>/<YOUR_USER_ID>/conda_envs/ascent/baseclone-ascent --clone base
+    $ source activate /ccsopen/proj/<YOUR_PROJECT_ID>/<YOUR_USER_ID>/conda_envs/ascent/baseclone-ascent
+    ```
+
+* Deleting an environment:
+
+    If for some reason you need to delete an environment, you can execute the following:
+
+    ```
+    $ conda env remove -p /path/to/your/env
+    ```
+
+* Adding known environment locations:
+
+    For a conda environment to callable by a "name", it must be installed in one of the `envs_dirs` directories.
```

Comment:
Here, I believe you meant to say
For a conda environment to "be" callable by ...